### PR TITLE
[8837] Open manual via default browser+added a field Handbook in Iatrix preferences

### PR DESCRIPTION
**Pull request by jamda**

### CODE REVIEW

File: ch.elexis.core.ui/src/ch/elexis/core/ui/actions/GlobalActions.java [hunk 1]
```diff
@@ -177,23 +178,25 @@ public void run(){
...
-								ExHandler.handle(e);
-							}
+					Desktop desktop = Desktop.getDesktop();
+					String url = "https://wiki.elexis.info";
+					
+					if(desktop != null)
+					{
+						try 
+						{
+							desktop.browse(new java.net.URI(url)); // Open default browser on system
```
Comment (col-panic) [inline ch.elexis.core.ui/src/ch/elexis/core/ui/actions/GlobalActions.java]: Please remove comments on obvious operations.

File: ch.elexis.core.ui/src/ch/elexis/core/ui/actions/GlobalActions.java [hunk 2]
```diff
@@ -177,23 +178,25 @@ public void run(){
...
+							logger.info("opening default browser." + url); // Added by Txomin
```
Comment (col-panic) [inline ch.elexis.core.ui/src/ch/elexis/core/ui/actions/GlobalActions.java]: Do not log this event. And if, only in trace or debug mode.

File: ch.elexis.core.ui/src/ch/elexis/core/ui/actions/GlobalActions.java [hunk 3]
```diff
@@ -177,23 +178,25 @@ public void run(){
...
+						}
+						catch (Exception e)
+						{
+							logger.info("failed to open default browser :" + e);
```
Comment (col-panic) [inline ch.elexis.core.ui/src/ch/elexis/core/ui/actions/GlobalActions.java]: set log level to warn

File: ch.elexis.core.ui/src/ch/elexis/core/ui/actions/GlobalActions.java [hunk 4]
```diff
@@ -177,23 +178,25 @@ public void run(){
...
+							ExHandler.handle(e);
 						}
+							
+					} else {
+						
+						logger.info("default browser not found.");
```
Comment (col-panic) [inline ch.elexis.core.ui/src/ch/elexis/core/ui/actions/GlobalActions.java]: misleading message. Desktop was null

File: ch.elexis.core.ui/src/ch/elexis/core/ui/actions/GlobalActions.java [hunk 5]
```diff
@@ -177,23 +178,24 @@ public void run(){
...
+							desktop.browse(new java.net.URI(url)); 
+						}
+						catch (Exception e)
+						{
+							logger.warn("failed to open default browser :" + e);
+							ExHandler.handle(e);
 						}
+							
+					} else {
+
```
Comment (col-panic) [inline ch.elexis.core.ui/src/ch/elexis/core/ui/actions/GlobalActions.java]: Please remove empty line, be sure to use our formatter profile https://github.com/elexis/elexis-3-core/blob/master/3lexisFormatterProfile.xml in your Eclipse workspace!

**Comment by col-panic** [comment]
Please provide a pull request against master, not 3.1 - patches always go in master first, and will then be backported respectively.

**Comment by col-panic** [comment]
Adapt the changes you  see in "Files changed" to the current master branch please!

**Comment by jamda** [comment]
Wich one is the current master branch?

**Pull request closed without merging**